src/lv_fonts/lv_font_template.c

```diff
@@ -0,0 +1,165 @@
+
+#include "lvgl/lvgl.h"
+
+/***********************************************************************************
+ * Size: 20 px
+ * Bpp: 1 (or 2/4/8/compressed)
+ * Fonts:
+ *  DejaVuSans.ttf:     U+0020 ( ) .. U+007e (~) range
```

Comment:
This can not be implemented without exact algorythm how to represent convertor options for multiple ranges and symbols. Great potential to spend a lot of time for nothing :)

IMO showing content of command line would be much better (at least, allow to reproduce with copy-paste), but it will not fit into single line (80 chars).

I suggest to drop this lines as not very useful.

Reply:
> IMO showing content of command line would be much better (at least, allow to reproduce with copy-paste), but it will not fit into single line (80 chars).

Perfect! Let's do it this way!